packages/component/src/Styles/StyleSet/CarouselFilmStrip.ts

```diff
@@ -19,6 +22,21 @@ export default function CarouselFilmStrip({
       },
 
       '& .webchat__carousel-filmstrip__attachment': {
+        '&:focus': {
```

Comment:
This is trickier in CSS BEM system.

In short, one element should not know about the position of another elements. In your CSS query, we assumed `webchat__carousel-filmstrip__attachment__focus-indicator` is placed under `webchat__carousel-filmstrip__attachment`. This "knowledge" should be avoided as much as we could.

FYI, block = `webchat__carousel-filmstrip`, element = `webchat__carousel-filmstrip__*`. Element should not know position of each other. They only know they are under the block.